src/css/main.scss

```diff
@@ -140,6 +171,11 @@ $blue-epsilon: #bce7f7;
 
 /* = WIZARD STEPS CONTAINER
 ----------------------------*/
```

Comment:
Bottom of the wizard arrow looks a bit off (I'll share a screenshot on slack).
Left hand side of wizard steps (rounded border of first item) looks weird after adding new border around whole container (plus sticking it to beginning of page).
Also arrows are looking odd on smaller resolutions (eg half of window on the 1360x768).